src/main/kotlin/com/example/pass_manager/web/dto/MongoClientDto.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.pass_manager.web.dto
+
+import jakarta.validation.constraints.NotBlank
+
+data class MongoClientDto(
```

Comment:
what if we replace mongo with mysql?
Then we should rename `MongoClientDto` -> `MysqlClientDto`?

Reply:
Renamed to ClientDto